integrations/test-gitlab-v2/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog - Ocean - test-gitlab-v2
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- towncrier release notes start -->
```

Comment:
you need to include the changelog notes. see how we've done it for other integrations